internal/converter/internal/otelcolconvert/testdata/servicegraph.river

```diff
@@ -0,0 +1,33 @@
+otelcol.receiver.otlp "default" {
+	grpc { }
+
+	http { }
+
+	output {
+		metrics = [otelcol.exporter.otlp.default.input]
```

Comment:
Is this correct to have the otlp receiver send to the otlp exporter directly? I'm trying to understand the concept of connectors in Otel so let me know.

Reply:
Ooh, I think this is a bug in the framework. The otlp receiver is not configured in any case to send metrics to the otlp exporter, so no, this isn't right. Nice catch.

Reply:
Whoops, will take a look, thanks for not glancing over it.

Reply:
This should be all fixed now